README.md

```diff
@@ -181,21 +181,15 @@ This should return the address of the contract on the testnet which looks like t
 └────────────────┴──────────────────────────────────────┴────────────────┘────────────────┘
 ```
 
-:::caution
-This scaffold comes with pre-configured faucet information for the testnet which is shared by all users and can cause issues. It is recommended that you replace the faucet info in the project's `config.json` file with your own whose setup instructions can be found [here](/docs/config/networks/#faucets). You can also get Tez for ghostnet from the [Teztnets Faucet](https://teztnets.xyz/).
-:::
+:warning: This scaffold comes with pre-configured faucet information for the testnet which is shared by all users and can cause issues. It is recommended that you replace the faucet info in the project's `config.json` file with your own whose setup instructions can be found [here](/docs/config/networks/#faucets). You can also get Tez for ghostnet from the [Teztnets Faucet](https://teztnets.xyz/).
 
 ### React Dapp
 
 The React dApp retrieves the number of available tacos from the smart contract and displays the value. It provides an interface for the user to buy tacos and looks like this:
 
 ![Hello Tacos Screenshot](/img/hello-tacos-screenshot.png)
 
```

Comment:
@alexzbusko the changes LGTM.

I know it is not part of this change but the `Hello Tacos Screenshot` is not loading when I view the file